packages/notifi-react-card/tsconfig.json

```diff
@@ -2,5 +2,6 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "jsx": "react",
+    "lib": ["ES2021", "DOM"]
```

Comment:
what effect does this have on tsup? just curious if ES2021 here will affect our compatibility (and by how much)

Reply:
I added support for ES2021 due to it impacting the `replaceAll` method. 

Would you recommend using something from here instead?
https://dmitripavlutin.com/replace-all-string-occurrences-javascript/

Reply:
does `tsup` stand for typescripti support?

Reply:
![Screen Shot 2022-10-04 at 11 00 35 AM](https://user-images.githubusercontent.com/100658137/193892371-168ee51a-4c9a-4a2a-88cd-1ba30c0fd350.png)

93% honestly feels good enough, particularly in the world of crypto where most users upgrade browsers often (for security purposes too)

I guess the main alternative would be a string.replace with regex?